.eslintrc

```diff
@@ -1,6 +1,8 @@
 {
+	"parser": "@typescript-eslint/parser",
 	"extends": "xo/esnext",
-	"ecmaFeatures": {
-		"globalReturn": true
+	"rules": {
+		"no-unused-vars": "off",
```

Comment:
We don't have to check this since TypeScript compiler does this.

Reply:
If porting to TypeScript, eslint should be removed and replaced with tslint.

Reply:
Recently Microsoft decided to throw away TSLint and focus on support on ESLint.  `@typescript-eslint` project is among that.

Reply:
Reference: https://github.com/Microsoft/TypeScript/issues/29288

Reply:
Oh, interesting. I was unaware

Reply:
Yeah, I was trying to put this info in Discord but been distracted.